docs/sources/mimir/operators-guide/mimir-runbooks/_index.md

```diff
@@ -476,6 +476,25 @@ How to **investigate**:
 
 - Look for details in the ingester logs
 
+### MimirIngesterInstanceHasNoTenants
+
+This alert fires when an ingester instance doesn't own any tenants and is therefore idling.
+
+How it **works**:
+
+- Ingesters join a hash ring that facilitates per-tenant request sharding across ingester replicas.
+- Distributors shard requests belonging to an individual tenant across a subset of ingester replicas. The number of replicas used per tenant is determined by the `-distributor.ingestion-tenant-shard-size` or the `ingestion_tenant_shard_size` limit.
+- When the tenant shard size is lower than the number of ingester replicas, some ingesters may not receive requests for any tenants.
```

Comment:
```suggestion
- When the tenant shard size is lower than the number of ingester replicas, some ingesters might not receive requests for any tenants.
```

Reply:
Rationale: Change spoken language to written language; disambiguate "may" (permission) from "might" (possibility).